docs/browser/README.md

```diff
@@ -15,6 +15,8 @@ Currently these tokens expire after 10hrs, this is not configurable yet.
 minio server /data
 ```
 
+This will mount the filesystem that Minio uses in the `/data` directory of the host machine. For example, if you run Minio locally and create a bucket called `foo`, then you can access any files you "uploaded" to the `foo` bucket by navigating to the `/data/foo` directory on your local disk.
```

Comment:
Minio doesn't mount anything, minio simply uses what is already present on the disk. Minio has other modes of operation as well so the line indicating accessing backend is not correct. In-fact it is discouraged to access the backend i.e /data directly even if its possible. 

Minio server manages concurrent access to backends - accessing backend directly would defeat this and lead to data loss and application issues.

Reply:
Interesting... My use case is to use minio as a scriptable front-end interface to my local filesystem, as I'm building an app on top of minio an alternative to using Finder.

Reply:
You can @billmei but this doc change might not be needed IMO.

Reply:
Ok, I will close this pull request.